node/src/api/accounts.rs

```diff
@@ -15,12 +15,15 @@ use aze_lib::storage::GameStorageSlotData;
 

```

Comment:
Please don't raise pr from develop. We will only raise pr from develop once the feature is quite well tested.